OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/util/ApplicationUtils.java

```diff
@@ -20,6 +20,7 @@
     private static final String HOME = System.getProperty("user.home");
     private static final String OS = System.getProperty("os.name");
     private static final String APP_DIR_NAME = ".OpenLaTeXStudio";
+    private static final String APP_VERSION = "1.4.0";
```

Comment:
Actually, I'd prefer that we do not hardcode the version here. We should make use of Maven properties and read the version from there (so we don't have to change version in multiple places in code each time the version changes). For the master branch, the version is set to development. You can find the property in pom.xml in the parent directory. There are a few approaches to reading maven properties, you can easily research them and pick the one that will work for us best.